packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -48,6 +48,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         { ...{ onBlurCapture: this._onTooltipMouseLeave } }
         onMouseEnter={ this._onTooltipMouseEnter }
         onMouseLeave={ this._onTooltipMouseLeave }
+        aria-describedby={ content }
```

Comment:
+1, we could use `getId` to give the tooltip a specific id, and then reference this here.